### PR TITLE
[16.0][IMP]helpdesk_mgmt: email channel

### PR DESCRIPTION
**Pull request by PicchiSeba**
Forward-ports https://github.com/OCA/helpdesk/pull/465 which itself forward-ported the commit https://github.com/OCA/helpdesk/commit/fcf0a41bbf4099d7926cf542402d23970a3dcf39 from 12.0 to 14.0

### CODE REVIEW

File: helpdesk_mgmt/models/helpdesk_ticket.py [hunk 1]
```diff
@@ -147,6 +147,12 @@ def _onchange_partner_id(self):
             self.partner_name = self.partner_id.name
             self.partner_email = self.partner_id.email
 
+    def _get_default_email_channel(self):
```
Comment (pedrobaeza) [inline helpdesk_mgmt/models/helpdesk_ticket.py]: Why creating this extra method?

Comment (PicchiSeba) [reply helpdesk_mgmt/models/helpdesk_ticket.py]: I can't say for sure as I'm not the original author, but my guess would be that this method was made to be overridable for custom email channels

Comment (pedrobaeza) [reply helpdesk_mgmt/models/helpdesk_ticket.py]: But the way to do that is with default values in the alias, and the other method is already prepared for that, so better to remove it. Let's not bloat the module.

Comment (PicchiSeba) [reply helpdesk_mgmt/models/helpdesk_ticket.py]: If you deem it unnecessary I can remove it

Comment (pedrobaeza) [reply helpdesk_mgmt/models/helpdesk_ticket.py]: Yes

Comment (pedrobaeza) [reply helpdesk_mgmt/models/helpdesk_ticket.py]: It's not removed yet.

Comment (PicchiSeba) [reply helpdesk_mgmt/models/helpdesk_ticket.py]: Yeah I'm sorry, I had just removed its use but not the method itself

**Review by flotho — APPROVED** [review]
LGTM, make sense

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge patch

**Comment by pedrobaeza** [comment]
Please fw-port it to 17

**Pull request merged**